payloads/library/execution/rename_entropy/payload.txt

```diff
@@ -0,0 +1,53 @@
+# Title: swap_jack
+# Description: swap_jack, simple yet powerful payload which can be used to replace one command with another on target’s system.
+# AUTHOR: drapl0n
+# Version: 1.0
+# Category: Execution.
+# Target: GNU/Linux.
+# Attackmodes: HID, STORAGE.
+
+LED SETUP
+ATTACKMODE STORAGE
```

Comment:
Your payload is doing keystroke injection, I would change this to `ATTACKMODE HID STORAGE` so the bash bunny can identify as both.